### DISCUSSION TITLE
Very cool

### DISCUSSION THREAD

**Discussion by ghost** [Verwex/Funkin-Mic-d-Up-SC#47] [General]
You have:
1. COMPLETELY changed the menu design, making it look like another game (none in particular, just doesn't look like funkin (in a good way))
2. Added NG recharts to desktop (I can't play on pre-rechart versions)
3. Added plenty more options
4. Added different game modes
5. Added different play modes

The team behind this are awesome!

**Comment by TentaRJ** [comment]
> You have:
> 
> 1. COMPLETELY changed the menu design, making it look like another game (none in particular, just doesn't look like funkin (in a good way))
> 2. Added NG recharts to desktop (I can't play on pre-rechart versions)
> 3. Added plenty more options
> 4. Added different game modes
> 5. Added different play modes
> 
> The team behind this are awesome!

There are more things to come :)

💚

**Reply by ghost** [reply]
Nice :)
